repository-data/webfiles/src/main/resources/site/freemarker/hee/layout/base-cookies-banner.ftl

```diff
@@ -0,0 +1,36 @@
+<#ftl output_format="HTML">
+<#include "../../include/imports.ftl">
+<@hst.defineObjects />
+<@hst.setBundle basename="uk.nhs.hee.web.global"/>
+<#assign gtmContainerId = "${hstRequestContext.resolvedMount.mount.channelInfo.GTMContainerID}">
+
+<div class="nhsuk-cookie-banner">
+    <#if showCookiesBanner>
+        <div class="nhsuk-width-container">
+            <h2 id="cookies0"><@fmt.message key="cookie.banner.title"/></h2>
+            <@fmt.message key="cookie.banner.text"/>
+            <ul>
+                <li><button class="nhsuk-button" id="nhsuk-cookie-banner__link_accept_analytics" tabindex="2"><@fmt.message key="cookie.button.accept"/></button></li>
+                <li><button class="nhsuk-button" id="nhsuk-cookie-banner__link_decline_analytics" tabindex="3"><@fmt.message key="cookie.button.reject"/></button></li>
+            </ul>
+        </div>
+    </#if>
+</div>
+
+<#if allowAnalyticsCookies>
```

Comment:
Could we join `<#if allowAnalyticsCookies>` & `<#if gtmContainerId?has_content>` like `<#if allowAnalyticsCookies && gtmContainerId?has_content>` for code optimisation ?

Reply:
Done.